.travis.yml

```diff
@@ -1,7 +1,7 @@
 node-preset: &node # Used internally to share configuration, ignored by travis.
   language: node_js
   node_js:
-    - "12"
```

Comment:
Nein. Our production env is on node 12. So we need to run our tests at least on node 12. We can run them on 12/14/16 if you want to